internal/rbd/rbd_attach.go

```diff
@@ -388,6 +388,33 @@ func appendKRbdDeviceTypeAndOptions(cmdArgs []string, userOptions string) []stri
 	return cmdArgs
 }
 
+// appendCrushLocationMapOptions parses crushLocationMap to the format
+// "--read_from_replica=localize,--crush_location=key1:value1|key3:value2"
+// and appends it to mapOptions.
+func appendCrushLocationMapOptions(mapOptions string, crushLocationMap map[string]string) string {
+	if len(crushLocationMap) == 0 {
+		// if no crush locations are found, return the mapOptions as it is.
+		return mapOptions
+	}
+
+	var b strings.Builder
+	b.WriteString("read_from_replica=localize,crush_location=")
```

Comment:
may be use the input string as `const` ?

Reply:
> may be use the input string as `const` ?

This currently used at only one place, let's do so when we'll be using it at multiple places.